ConnectionManager.lib.nut

```diff
@@ -1,15 +1,36 @@
+// The MIT License (MIT)
```

Comment:
Can you please use the license file and headers from https://electricimp.com/docs/libraries/submissions/?